### PR TITLE
[patch] Include prompt for mongodb namespace

### LINKED ISSUE #301: Add option to select name of common namespaces
We would like to be able to select a name for the following namespaces since we think that "grafana" or "mongoce" is too generic and would be difficult to associate with MAS in an Openshift/Kubernetes cluster full of applications:
```
{{ lookup('env', 'DB2_NAMESPACE') | default('db2u', true) }}
{{ lookup('env', 'GRAFANA_NAMESPACE') | default('grafana', true) }}
{{ lookup('env', 'MONGODB_NAMESPACE') | default('mongoce', true) }}
```

Thanks!

### PR DESCRIPTION
**Pull request by caroazad**
This is a follow up on this issue #301 to add mongodb namespace prompt. The other part of the work was merged on PR https://github.com/ibm-mas/cli/pull/319

### CODE REVIEW

File: image/cli/mascli/functions/install [hunk 1]
```diff
@@ -18,6 +18,9 @@ Entitlement & Licensing (Required):
       --uds-firstname ${COLOR_YELLOW}UDS_CONTACT_FIRSTNAME${TEXT_RESET}                        Contact first name
       --uds-lastname ${COLOR_YELLOW}UDS_CONTACT_LASTNAME${TEXT_RESET}                          Contact last name
 
+Mongo DB (Optional):
```
Comment (andrercm) [inline image/cli/mascli/functions/install]: Can you standardize the naming convention to read "MongoDB", please?

File: image/cli/mascli/functions/pipeline_config [hunk 2]
```diff
@@ -218,6 +218,10 @@ function pipeline_config() {
   fi
   SLS_LICENSE_FILE="/workspace/entitlement/$(basename $SLS_LICENSE_FILE_LOCAL)"
 
+  echo
+  echo_h2 "Configure Mongodb"
```
Comment (andrercm) [inline image/cli/mascli/functions/pipeline_config]: Can you standardize the naming convention to read "MongoDB", please?

File: image/cli/mascli/functions/pipeline_config [hunk 3]
```diff
@@ -218,6 +218,10 @@ function pipeline_config() {
...
+  prompt_for_input "Mongodb Namespace" MONGODB_NAMESPACE "mongoce"
```
Comment (andrercm) [inline image/cli/mascli/functions/pipeline_config]: Same as above

Comment (andrercm) [reply image/cli/mascli/functions/pipeline_config]: @caroazad can you also add an entry for MongoDB namespace in here:
https://github.com/ibm-mas/cli/blob/common-namespace/image/cli/mascli/functions/pipeline_show_config#L106

**Review by andrercm — CHANGES_REQUESTED** [review]
just few minor adjustments

**Review by andrercm — CHANGES_REQUESTED** [review]
just few minor adjustments

**Review by durera — COMMENTED** [review]
Could we put all the namespace configuration into the optional Advanced settings section?  99.999% of people don't care/don't need to care and want to avoid the interactive mode becoming a chore to work through accepting defaults, the main idea is still to have this be a quick/ppainless way to install MAS and keep people "on rails" as much as possible.

See: `image/cli/mascli/functions/pipeline_config_advanced`

**Review by durera — APPROVED** [review]

**Pull request merged**